_topic_maps/_topic_map.yml

```
@@ -2813,6 +2813,23 @@ Topics:
       File: logging-5-8-release-notes
     - Name: Logging 5.7
       File: logging-5-7-release-notes
+  - Name: Logging 6.0
+    Dir: logging-6.0
+    Topics:
+      - Name: Release notes
+        File: log6x-release-notes
+      - Name: About logging 6.0
+        File: log6x-about
+      - Name: Upgrading to Logging 6.0
+        File: log6x-upgrading-to-6
+      - Name: Configuring log forwarding
+        File: log6x-clf
+      - Name: Configuring LokiStack storage
+        File: log6x-loki
+      - Name: Visualization for logging
```

Does this need to be capitalized?

```suggestion
      - Name: Visualization for Logging
```

---

No. Our official product name is "logging for Red Hat OpenShift", our approved shortname is "logging", and headings are sentence case per the style guide.